src/App.tsx

```diff
@@ -239,6 +244,45 @@ const Profile = ({ user, userData, userName }) => {
   );
 };
 
+interface TrustLevelProperties {
+  trustLevel: number;
+  networkTime: number; //time on network in days
+}
+
+class TrustLevel extends React.Component<TrustLevelProperties, {}> {
+
+  render() {
+    var trustLevel = this.props.trustLevel;
+    var networkTime = this.props.networkTime;
+    return (
+      <div className="Trust">
+          <div>Trust Level: {trustLevel}</div>
+          { this.computeTrustSuggestion(trustLevel, networkTime) }
+      </div>
+    );
+  }
+
+// TODO: Based on the FAQ trust is determined by:
+// 1. Time on the network
+// 2. Number of Level 2 and 3 accounts that trust this account
+// 3. Level 3 verification video in last month || invited in first month
+computeTrustSuggestion = (trustLevel, networkTime) => {
+  switch (trustLevel) {
+      case 0:
+      case 1:
+      case 2:
+        if (networkTime < 30 ) {
+           return <div> Stay active on the network for longer to increase your trust level </div>;
+        }
+        return <div> increase your trust receiving trust from level 2 or 3 accounts </div>;
```

Comment:
I like this bit about telling users what they need to level up! Can pull more deets from FAQ